### PR TITLE
Stratified sampling

### PR DESCRIPTION
**Pull request by ChuckHend**
Take 2 of https://github.com/postgresml/postgresml/pull/1290

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -58,8 +63,13 @@ jobs:
             cargo pgrx init
           fi
 
+          git checkout master
+          echo "\q" | cargo pgrx run
+          psql -p 28816 -h localhost -d pgml -P pager -c "CREATE EXTENSION pgml;"
+          git checkout $CI_BRANCH
+          echo "\q" | cargo pgrx run
+          psql -p 28816 -h localhost -d pgml -P pager -c "ALTER EXTENSION pgml UPDATE;"
```
Comment (ChuckHend) [inline .github/workflows/ci.yml:71]: intention for this is to just validate that `alter extension update` actually runs

**Comment by montanalow** [comment]
I verified with tests locally, because I wasn't able to figure out how to get github actions running on this PR... weird 🤷 .

**Comment by ChuckHend** [comment]
> I verified with tests locally, because I wasn't able to figure out how to get github actions running on this PR... weird 🤷 .

Maybe had something to do with how I had this PR open from `master` on my fork, then I closed it and re-opened from a different branch from my fork. My bad :(

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -58,8 +63,13 @@ jobs:
             cargo pgrx init
           fi
 
+          git checkout master
+          echo "\q" | cargo pgrx run
+          psql -p 28816 -h localhost -d pgml -P pager -c "CREATE EXTENSION pgml;"
+          git checkout $CI_BRANCH
+          echo "\q" | cargo pgrx run
```
Comment (levkk) [inline .github/workflows/ci.yml:70]: You can use `cargo pgrx start`

**Pull request merged**